src/workspacer.Shared/Workspace/Workspace.cs

```diff
@@ -73,7 +73,15 @@ public void AddWindow(IWindow window, bool layout = true)
                     _lastFocused = window;
                 }
 
-                _windows.Add(window);
+                if (_context.NewWindowAsMaster)
+                {
+                    _windows.Insert(0, window);
+                }
+                else
+                {
+                    _windows.Add(window);
+                }
```

Comment:
This change looks simple enough, and from what I can tell changes the order of which windows gets added.

How about, instead of a boolk, making this an enum? Then one can make both the property-name and property-value more descriptive.

A quick suggestions here, without too much thought into it (yet):

````c#
WindowOrder WindowAddingOrder {get; set; }

public enum WindowOrder
{
    NewWindowsLast, // default
    NewWindowsFirst
}
````

Wouldn't that be a more clear description of what this change involves?

Reply:
maybe `WindowOrder => NewWindowOrder` but +1